src/main/java/de/leonhard/storage/Config.java

```diff
@@ -56,6 +58,34 @@ public Config(final File file) {
     super(file);
   }
 
+  public void annotateClass(Object classInstance) {
+    Class<?> clazz = classInstance.getClass();
+    try {
+      for (Field field : clazz.getFields()) {
+        if(field.isAnnotationPresent(ConfigPath.class)) {
+          field.setAccessible(true);
+          field.set(classInstance, this.get(field.getAnnotation(ConfigPath.class).value(), field.getType()));
+        }
+      }
+    }catch (IllegalAccessException e) {
+      e.printStackTrace();
```

Comment:
Just printing the exception is not that good in my opinion, maybe logging it properly or letting the caller decide how to react would be better practice (maybe not with an Exception but a RuntimeException?)